tests/test_parse.py

```diff
@@ -16,10 +15,7 @@
 from   pyflyby._imports2s       import SourceToSourceFileImportsTransformation
 
 
-if sys.version_info < (3, 8, 3):
-    print_function_flag = CompilerFlags.from_int(0x10000)
-else:
-    print_function_flag = CompilerFlags.from_int(0x100000)
+print_function_flag = CompilerFlags.from_int(0x100000)
 
```

Comment:
Over eager in removing ? 3.8.0-3.8.2 are still supported I think. Better be safe ?

```suggestion
if sys.version_info < (3, 8, 3):
    print_function_flag = CompilerFlags.from_int(0x10000)
else:
    print_function_flag = CompilerFlags.from_int(0x100000)
```